pkg/ccl/backupccl/manifest_handling.go

```diff
@@ -1037,23 +1037,57 @@ func sanitizeLocalityKV(kv string) string {
 	return string(sanitizedKV)
 }
 
+// readEncryptionOptions takes in a backup location and tries to find and return all encryption option files in the
+// backup. A backup normally only creates one encryption option file, but if the user uses ALTER BACKUP to add new
+// keys, a new encryption option file will be placed side by side with the old one. Since the old file is still valid,
+// as we never want to modify or delete an existing backup, we return both new and old files.
```

Comment:
nit: we usually wrap comments to 70 characters, i think you can configure goland to do this

Reply:
here and the few others places you have long comments in this PR